### PR TITLE
Attributes parsed correctly when newline precedes attributes

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #484 
@NTaylorMullen @dougbu

### CODE REVIEW

File: src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs [hunk 1]
```diff
@@ -405,9 +405,9 @@ private bool IsTagRecoveryStopPoint(HtmlSymbol sym)
 
         private void TagContent()
         {
-            if (!At(HtmlSymbolType.WhiteSpace))
+            if (!At(HtmlSymbolType.WhiteSpace) && !At(HtmlSymbolType.NewLine))
```
Comment (dougbu) [inline src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs]: Nice. Do we have other cases where a newline would confuse the parser? For example does `EndTagBlock()` in this class handle the following?
``` html
   <p
    >
```

Comment (dougbu) [inline src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs]: Suggest checking around for uses of `HtmlSymbolType.WhiteSpace` alone that seem suspect. Then test a bit.

**Comment by dougbu** [comment]
:shipit: but please confirm no other evil is lurking in nearby code.

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request merged**